packaging/examples/metrics/grafana-dashboards/strimzi-kraft.json

```diff
@@ -329,7 +329,7 @@
       "targets": [
         {
           "datasource": "${DS_PROMETHEUS}",
-          "expr": "sum(kubelet_volume_stats_available_bytes{namespace=\"$kubernetes_namespace\",persistentvolumeclaim=~\"data(-[0-9]+)?-$strimzi_cluster_name-(kafka|$pool_name)-[0-9]+\"}) by (persistentvolumeclaim)",
+          "expr": "sum(kubelet_volume_stats_available_bytes{namespace=\"$kubernetes_namespace\",persistentvolumeclaim=~\"data(-[0-9]+)?-$strimzi_cluster_name-$kafka_broker\", persistentvolumeclaim!~\".*zookeeper(-[0-9]+)?$\"}) by (persistentvolumeclaim)",
```

Comment:
I gave this a try. The KRaft dashboard is not using `$kafka_broker` but `$kraft_node`. Could you please fix this?

Reply:
(the same is of course also in the Helm Chart dashboard)

Reply:
> I gave this a try. The KRaft dashboard is not using `$kafka_broker` but `$kraft_node`. Could you please fix this?

Changed, please help to try again